docs/topics/BrowserPlugin.adoc

```diff
@@ -138,6 +138,22 @@ WARNING: We do not recommend changing any of these settings as they may break th
 image::browser_advanced_settings.png[]
 
 === Advanced Setup
+
+==== Custom Browser option
+
+It is possible to enable support for a custom browser (e.g. Librefox, Arc, etc.) using this feature.

```

Comment:
Make a note that this is only available on Linux and macOS. Maybe also mention "beta and nightly browsers" and "Waterfox"